src/math-translator.js

```diff
@@ -40,12 +52,39 @@ const MATH_RULES_LOCALES = {
     CSC_AS_COSSEC: ['pt', 'pt-pt'],
     // Rules conditional on the translated template
     MAYBE_DIV_AS_COLON: ['id', 'lol'],
-    MAYBE_TIMES_AS_CDOT: ['az', 'bn', 'el', 'hi', 'hy', 'id', 'it', 'ja', 'ka',
+    MAYBE_TIMES_AS_CDOT: ['az', 'bn', 'el', 'hi', 'id', 'it', 'ja', 'ka',
             'ko', 'nl', 'pt', 'ru', 'uk', 'zh-hans', 'lol'],
-    MAYBE_CDOT_AS_TIMES: ['az', 'bn', 'el', 'hi', 'hy', 'id', 'it', 'ja', 'ka',
+    MAYBE_CDOT_AS_TIMES: ['az', 'bn', 'el', 'hi', 'id', 'it', 'ja', 'ka',
             'ko', 'nl', 'pt', 'ru', 'uk', 'zh-hans', 'lol'],
 };
 
+
+/**
+ * Translates math notation in English strings to match
+ * notation for a given language, such as thousand separators,
+ * decimal commas, math operators etc.
+ *
+ * This function serves as an interface to all the other functions
```

Comment:
I moved this function up in the file. Otherwise, the only change here is that we now call `maybeTranslateMath()` from within this function.